scripts/multinode/configure-sec.sh

```diff
@@ -109,6 +117,49 @@ EOF
         --cluster-domain=cluster.local
 }
 
+function configure_node() {
+    local service_ready=false
+
+    echo "Waiting for MicroShift nodes to be ready"
+    for _ in $(seq 300) ; do
```

Comment:
Is this equivalent to `oc wait --for=condition=Ready nodes --all --timeout=300s`?

Reply:
I still have to add the loop because the command fails initially because `microshift-sec` takes time to be created.

Reply:
For the same reason, I have to check each node separately, or I'm getting errors. 
I enhanced the checks though. Is this better?